src/SingleArticle.jsx

```diff
@@ -1,22 +1,43 @@
 import { useState, useEffect } from "react";
+import { useParams } from 'react-router-dom';
 import { getSingleArticle } from "./utils/api";
+import { format, parseISO } from 'date-fns'
+import './SingleArticle.css'
 
-const ArticleList = () => {
-
+const SingleArticle = () => {
+const {article_id} = useParams();
```

Comment:
Great use of useParams to ensure this page works if a user goes directly to the article 👍